assets/custom.js

```diff
@@ -10,8 +10,16 @@ function activate_sidebar() {
 function activate_dropdown() {
 	// body...
 	$(document).ready(function(){
-		$('.ui.selection.dropdown').click(function(){
-			$('.ui.dropdown').dropdown('show');
+		$('#project_filter').click(function(){
+			$('#project_filter').dropdown();
+		});
+
+		$('#feed1').click(function(){
+			$('#feed1').dropdown();
```

Comment:
add a class to any object which decided to have dropdown and initiate your dropdown using a class.

example `$('.with-dropdown').dropdown()` so any classes with `.with-dropdown` will have dropdown feature.